src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+@import './variables';
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: $star-size;
+    height: $star-size;
```

Comment:
Ensure that the variable `$star-size` is defined in the 'variables' file. Missing variable definitions will lead to compilation errors.